src/Shared/FileSystem/IFileSystem.cs

```diff
@@ -6,39 +6,39 @@
 
 namespace Microsoft.Build.Shared.FileSystem
 {
-    /// <summary>
-    /// Abstracts away some file system operations
-    /// </summary>
+
+    /*
+     * This is a clone of Microsoft.Build.FileSystem.IMSBuildFileSystem.
```

Comment:
I noticed a reference to IFileSystem earlier. Made me wonder if it would be worth it to have something that detected references to "deprecated" APIs and gave a warning about them.

Reply:
IFileSystem will remain the internal interface that MSBuild uses until we can break that old nuget version.